build.gradle

```diff
@@ -95,6 +96,11 @@ dependencies {
     implementation 'jstl:jstl:1.2'
     // http://commons.apache.org/proper/commons-configuration/dependencies.html
     implementation 'commons-beanutils:commons-beanutils:1.9.3'
+    implementation 'com.zaxxer:HikariCP:3.3.1'
+    // https://mvnrepository.com/artifact/org.flywaydb/flyway-core
+    implementation group: 'org.flywaydb', name: 'flyway-core', version: '7.2.0'
+    //This takes care of 'TestContainer' versioning
+    implementation platform('org.testcontainers:testcontainers-bom:1.15.3') //import bom
```

Comment:
Again, this would better be `testImplementation`.

Also, be careful about overcommenting. `//import bom` is precisely what the line is doing, it adds no information. I remember one textbook example which was something like this:
```c
return a + b;  // Add a and b and return the sum
```

Reply:
Should be `testImplementation` Sorry, copy-paste error. Will take care.